src/pyxplor/plot_categorical.py

```diff
@@ -1,55 +1,139 @@
 import pandas as pd
+import math
+import matplotlib.pyplot as plt
+import seaborn as sns
 
 def plot_categorical(
         input_df: pd.DataFrame,
         list_of_variables: list,
-        label_y_offset: int=0,
         label_fontsize: int=10,
```

Comment:
We need to decide whether we should have a default value for these arguments. I didn't set a default because I thought that these arguments needs to be adjusted if the figsize is vastly different. But I guess it doesn't hurt to add it.